lib/pacemaker/pcmk_sched_utils.c

```diff
@@ -715,10 +715,6 @@ pcmk__create_history_xml(xmlNode *parent, lrmd_event_data_t *op,
                 crm_xml_add_ll(xml_op, XML_RSC_OP_LAST_CHANGE,
                                (long long) op->t_run);
 
-                // @COMPAT last-run is deprecated
-                crm_xml_add_ll(xml_op, XML_RSC_OP_LAST_RUN,
-                               (long long) op->t_run);
-
```

Comment:
last-run is still mentioned in the docs and all over the place in CTS.  Maybe it makes sense to get rid of it in those places too, or at least add a note that it's deprecated to Pacemaker Explained.

Reply:
Removed, except for in scheduler inputs, since the code is supposed to be able to handle old saved files